### ISSUE TITLE
Missing method?

### ISSUE THREAD

**Issue by simonbray** [InformaticsMatters/pipelines-utils-rdkit#5]
Hi, the `filter.fragment` method used here (https://github.com/InformaticsMatters/pipelines/blob/master/src/python/pipelines/rdkit/screen.py#L102) does not seem to exist. Running the script results in an ImportError.

Was it available in a previous version of this library?

Simon

ping @lorrainealisha75

**Comment by tdudgeon** [comment]
Hi, yes, that method was moved. I've fixed this now: https://github.com/InformaticsMatters/pipelines/commit/d90125b5847ff0238c48a0e259c4b8ce3fc183f1

@alanbchristie could you push the changes to PyPi?

**Comment by simonbray** [comment]
The conda package which we are using is actually built from the GitHub repository rather than PyPI. (See here: https://github.com/conda-forge/im-pipelines-utils-rdkit-feedstock/blob/c7e555f5ed51b71d7924c551205ce5c7cd7de716/recipe/meta.yaml#L9.) So it would be great if a new release could be made on GitHub in addition to the PyPI release.

**Comment by alanbchristie** [comment]
Hi,

As far as I can see the fix to the pipelines only impacts the informaticsmatters/pipelines repository with Tim’s commit today which adjusts the screen.py pipeline implementation. There’s nothing new I can see for PyPi so the existing PyPi reference should work.

As I’ve not been involved in this topic before, what needs to be done to "make a new release on GitHub” for the pipelines repo other than provide the commit tag (d90125b5847ff0238c48a0e259c4b8ce3fc183f1)?

Ideally we should be creating tags and releases in the pipelines repo but we haven’t done this yet.

----
Alan

**Comment by simonbray** [comment]
Ah sorry, you're right, there was no change to this repository. So nothing needs to be updated here, since the latest release is in sync with the latest commit.

**Comment by alanbchristie** [comment]
OK, so there’s nothing more you need me to do?

I’ll discuss strategies for tagging our pipeline repositories going forward, for now we tend to assume the latest ‘master’ is a workable release.

Alan Christie
achristie@informaticsmatters.com

**Comment by simonbray** [comment]
No, I don't think so. At least, not at the moment. It would be nice to have official releases of the pipelines repo though.

**Comment by simonbray** [comment]
Another question: looking at the one of the tests (here: https://github.com/InformaticsMatters/pipelines/blob/master/test-rdkit.sh#L9)

```
python src/python/pipelines/rdkit/screen.py
  --qsmiles 'C1N=C(C2=CC=CC=C2)C2=CC=CC=C2C2=C1C=NC(NC1=CC=CC=C1)=N2'
  --simmin 0.45
  -if sdf
```

Is there a reason that a SMILES string is given, but the input format is `sdf`? Or is there a more up-to-date version of the tests somewhere else?

**Comment by tdudgeon** [comment]
@simonbray The query structure is specified as smiles, but the molecules to be screened come from a SD file.

**Comment by simonbray** [comment]
Makes sense, thanks Tim.
